mllib-local/pom.xml

```diff
@@ -62,6 +62,15 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.json4s</groupId>
+      <artifactId>json4s-jackson_${scala.binary.version}</artifactId>
+      <version>3.2.10</version>
```

Comment:
Core also depends on json4s (See https://github.com/apache/spark/blob/master/core/pom.xml#L195.). Shall we move the version definition to a common place?